modules/shortcodes/dailymotion.php

```diff
@@ -327,3 +327,61 @@ function jetpack_dailymotion_embed_reversal( $content ) {
 }
 
 add_filter( 'pre_kses', 'jetpack_dailymotion_embed_reversal' );
+
+/**
+ * DailyMotion Widget Shortcode
+ *
+ * Example:
+ * [dailymotion-widget id=XYZ]
+ *
```

Comment:
please add the following to all the phpdocs in this file:
```php
* @module shortcodes
```
For reference, [see this](https://github.com/Automattic/jetpack/blob/master/modules/shortcodes/vimeo.php#L115).